airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -624,8 +629,12 @@ def __init__(
         self._attempts = 10
         self._attempt_interval = 15
         self.redshift_hook = RedshiftHook(aws_conn_id=aws_conn_id)
+        self.aws_conn_id = aws_conn_id
+        self.deferrable = deferrable
+        self.max_attempts = max_attempts
```

Comment:
How is this different to `self._attempts`? max_attempts is currently ignored in sync mode and it probably shouldn't be.

Reply:
self._attempts: we added this because sometimes redshift goes into an unexpected state which does not allow us to perform the other operation cluster so this param allows us to wait for the right cluster state.
self.max_attempts: once delete is initiated how many times we should poll to check the status before timeout.